homeassistant/components/abode/__init__.py

```diff
@@ -179,27 +179,27 @@ def capture_image(call):
         """Capture a new image."""
         entity_ids = call.data.get(ATTR_ENTITY_ID)
 
-        target_devices = [
-            device
-            for device in hass.data[DOMAIN].devices
-            if device.entity_id in entity_ids
+        target_entities = [
+            entity
+            for entity in hass.data[DOMAIN].entities
+            if entity.entity_id in entity_ids
         ]
 
-        for device in target_devices:
-            device.capture()
+        for entity in target_entities:
+            entity.capture()
```

Comment:
Here we want to send the signal targeting the entity_id with `dispatcher_send`.

Reply:
This will take me some time to figure out - I am unfamiliar with the dispatch helper, so I'm not quite sure what this means yet. :) I'll keep looking at examples.